src/components/Fields/Extensions.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+
+import { OptionsContext } from '../OptionsProvider';
+
+import { SchemaModel } from '../../services/models';
+import { FieldDetail } from './FieldDetail';
+
+export interface ExtensionsProps {
+  schema: SchemaModel;
+}
+
+export class Extensions extends React.PureComponent<ExtensionsProps> {
+  redocExtensions = [
```

Comment:
Could you move this array to https://github.com/Rebilly/ReDoc/blob/master/src/utils/openapi.ts

Also, having it as an object would be a bit faster than `array.includes` lookup.